### DISCUSSION TITLE
Concatinating Array's with non-identical items

### DISCUSSION THREAD

**Discussion by ghost** [neuefische/questions#481] [Web] [answered]
Lets assume I have 2 Arrays:

```
const arrayOne = [1, 2, 3]
const arrayTwo = [3, 4, 5]
```

I want to add them together, but without double items!
With the rest operator of course I get:

```
const addedArray = [...arrayOne, ...arrayTwo] = [1, 2 ,3 ,3 ,4 ,5]
```

I want a final array that looks like this:
```
const wantedArray = [1, 2, 3, 4, 5]
```

Is there a good array method to achieve this goal?

**Comment by ThomasF85** [comment, marked as answer]
Hi Tariq,

you can use JavaScript Sets and the spread syntax to achieve your goal:
```
const wantedArray = [...new Set([...arrayOne, ...arrayTwo])]
```
